src/main/webapp/WEB-INF/tags/student/studentMessageOfTheDay.tag

```diff
@@ -4,24 +4,9 @@
 <c:set var="motdUrl" value="<%= Config.inst().STUDENT_MOTD_URL %>" />
 <c:if test="${not empty motdUrl}">
     <script>
-        var url = window.location.origin + '/' + '<c:out value="${motdUrl}" />';
-
-        $.ajax({
-            type: 'GET',
-            url: url,
-            success: function(data) {
-                $('#student-motd').html(data);
-            },
-            error: function(jqXHR, textStatus, errorThrown) {
-                console.log('AJAX request failed');
-            }
-        });
-
-        function closeMotd() {
-            $('#student-motd-container').hide();
-        }
+        var motdUrl = window.location.origin + '/<c:out value="${motdUrl}" />';
     </script>
-
+    <script type="text/javascript" src="/js/studentMotd.js"></script>
```

Comment:
I have a memory of seeing somewhere that js files are loaded in the order they are included. Any chance the script may be called before it is loaded? Does the browser wait till all js files are loaded before it renders the page?